domain/schema/model/sql/0019-unit.sql

```diff
@@ -120,26 +120,27 @@ CREATE TABLE unit_state_relation (
     REFERENCES unit (uuid)
 );
 
+-- cloud containers belong to a k8s unit.
 CREATE TABLE cloud_container (
-    -- one cloud container per net node.
-    net_node_uuid TEXT NOT NULL PRIMARY KEY,
+    unit_uuid TEXT NOT NULL PRIMARY KEY,
     -- provider_id comes from the provider, no FK.
+    -- it represents the k8s pod UID.
     provider_id TEXT NOT NULL,
-    CONSTRAINT fk_cloud_container_net_node
-    FOREIGN KEY (net_node_uuid)
-    REFERENCES net_node (uuid)
+    CONSTRAINT fk_cloud_container_unit
+    FOREIGN KEY (unit_uuid)
+    REFERENCES unit (uuid)
 );
 
 CREATE UNIQUE INDEX idx_cloud_container_provider
 ON cloud_container (provider_id);
 
 CREATE TABLE cloud_container_port (
-    cloud_container_uuid TEXT NOT NULL,
+    unit_uuid TEXT NOT NULL,
     port TEXT NOT NULL,
-    CONSTRAINT fk_cloud_container_port_net_node
-    FOREIGN KEY (cloud_container_uuid)
```

Comment:
This here was actually my main point of contention - that a column `cloud_container_uuid` was equivalent to `net_node_uuid` in the other table.

Reply:
The intent here was to convey the concept that the container port is related to a container. The fact that the container is tied to a net node is somewhat irrelevant to *this* port -> container relation. Conceptually the net_node_uuid in the cloud_container table is equivalent to a cloud_container_uuid since it's the primary key. I've seen plenty of data modelling where this sort of thing is done. I'm guessing we're looking to avoid it.